graphs/src/index.html

```diff
@@ -7,6 +7,7 @@
    <script type="application/javascript" src="/graphs/metrics.js"></script>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta http-equiv="refresh" content="3600">
```

Comment:
Why are you refreshing? It would surprise me that after an hour the pages suddenly refreshes

Reply:
The idea is that at some point we can have this site on a screen unattended. I figured updating the data once every hour is enough to sho 'live' data.